src/bench/binary-operation-benchmark.cpp

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
Needs to be moved to `src/bench/binary` to have a folder per libgdf operator (one folder for binary, one for parquet, one for joins, etc.)

Reply:
https://github.com/BlazingDB/libgdf/tree/binary-operators-draft/src/bench/binary-operation